examples/demo-app/src/constants/default-settings.js

```diff
@@ -66,7 +67,24 @@ export const LOADING_METHODS = [
     options: [
       // Dynamically populated
     ]
+  },
+  {
+    id: 'sharedstreets',
+    label: 'Load SharedStreets Data'
   }
 ];
 
 export const DEFAULT_LOADING_METHOD = LOADING_METHODS[0];
+
+export const SHAREDSTREETS_DATASETS = [
+  {
+    id: 'pickup-dropoff',
+    configUrl: "",
+    dataUrl: "",
+    description: "Pick up and drop off data",
+    imageUrl: "https://s3.amazonaws.com/sharedstreets-samples/pickup-dropoff.png",
```

Comment:
- Add proper color and radius to the screenshot
- This image is 2.4MB. A 400x300 jpg is probably enough.